rfc/0002-subdocapi.md

```diff
@@ -179,11 +208,12 @@ Be aware that for lookup operations which fail partially in languages which can
 These errors are only received for `counter_in` operations:
 
 * NumberTooBig: Existing number value in document is too big (out of range between `INT64_MIN` and `INT64_MAX`
-* ZeroDelta: Got a zero value for a delta
+* BadDelta: This corresponds to the `PROTOCOL_BINARY_RESPONSE_DELTA_ERANGE`
```

Comment:
Isn't there also the case where the delta, applied to the current value, makes the counter too big?
Note that in a strongly-typed language like Java, only this case above and the "delta is 0" one can happen. Delta must be a number and cannot be itself too large by way of the type being `long` or equivalent (64bits signed integer).